codalab/lib/codalab_manager.py

```diff
@@ -113,14 +114,15 @@ class CodaLabManager(object):
     '''
     temporary: don't use config files
     '''
-    def __init__(self, temporary=False, clients=None):
+    def __init__(self, temporary=False, clients=None, rest_clients=None):
```

Comment:
This class is a giant mess and you've made it worse. See if you can make it cleaner.